ui/app/helpers/confirm-transaction/util.js

```diff
@@ -0,0 +1,116 @@
+import currencyFormatter from 'currency-formatter'
```

Comment:
I asked a similar question elsewhere. Your thoughts on including this in a general ` ui/app/helpers/` directory as opposed to within a root directory for confirm-transaction components?

Reply:
I just found myself reusing these in various components and the .duck, and it got annoying having to move these functions/files around based on which component was using a function during debugging/refactoring (up one level when used by a parent, down one level when not used).